src/app/components/ConfirmDelete/index.js

```diff
@@ -15,10 +15,10 @@ const ConfirmDelete = ({ onClickAcceptHandler, onClickRejectHandler, item }) =>
         <h3 className={ style.confirmDeleteThirdHeading }>Are you certain?</h3>
       </div>
       <div className={ style.confirmDeleteButtons }>
-        <button onClick={ onClickRejectHandler } className={ `${style.confirmDeleteButton} ${style.confirmDeleteReject}` }>
+        <button onClick={ onClickRejectHandler } className={ style.confirmDeleteButton + ' ' + style.confirmDeleteReject }>
```

Comment:
We have an NPM package installed to handle this named `classnames`
The code would look like this `className={classNames(style.confirmDeleteButton, style.confirmDeleteReject)}`

Reply:
What does the package do behind the scenes? If it works for @phetter I'm ok with it.